sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -1,30 +1,114 @@
 # Azure Container Registry client library for .NET
 
-Azure Container Registry allows you to build, store, and manage container images and artifacts in a private registry for all types of container deployments.
+Azure Container Registry allows you to store and manage container images and artifacts in a private registry for all types of container deployments.
 
 Use the client library for Azure Container Registry to:
 
-- List the repositories in a registry
-- List the images or artifacts in a repository
+- List images or artifacts in a registry
+- Obtain metadata for images and artifacts, repositories and tags
+- Set read/write/delete properties on registry items
+- Delete images and artifacts, repositories and tags
 
-  [Source code](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/containerregistry/Azure.Containers.ContainerRegistry) | Package (NuGet) | API reference documentation | [Product documentation](https://docs.microsoft.com//azure/container-registry)
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
 ## Getting started
 
 ### Install the package
 
+Install the Azure Container Registry client library for .NET with [NuGet][nuget]:
+
+```Powershell
+dotnet add package Azure.Containers.ContainerRegistry
```

Comment:
I believe you'll need a `--version` to specify the pre-release version, or use `--prerelease`